guevaraTorres/assignments/fastSlow/lc141/lc141.js

```diff
@@ -11,6 +20,24 @@ class Node {
 
 const has_cycle = function(head) {
   // TODO: Write your code here
+
+  let slow = head,
+  fast = head;
+
+  while(fast !== null || fast.next !== null ){
+    if(fast.next == slow) return true;
+    slow = slow.next;
+    fast = fast.next.next;
+  }
```

Comment:
Interesting if statement, although I am curious why you are comparing the node at fast.next to the node at slow instead of the node at fast.